rules/codelingo/effective-go/avoid-annotations-in-comments/codelingo.yaml

```diff
@@ -27,14 +27,14 @@ tenets:
           The package comment for the fmt package uses this to good effect.
```

Comment:
I reckon it'd be good to add a hyperlink to the `fmt` package here. https://golang.org/pkg/fmt/